class03/exercise_7.py

```diff
@@ -31,9 +32,20 @@ def check_path(path):
 	# if doesn't exist
 	>>> False, []
 	'''
-
-
-def read_csv(file):
+	import os
```

Comment:
common practice is to put all imports at the very begeining of the file (e.g. line 6 )